.travis.yml

```diff
@@ -11,7 +11,6 @@ language: python
 python: "2.7"
 
 env:
-  - ANSIBLE_INSTALL_VERSION=1.9.4
   - ANSIBLE_INSTALL_VERSION=2.0.0.2
   - ANSIBLE_INSTALL_VERSION=2.0.2.0
   - ANSIBLE_INSTALL_VERSION=2.1.1.0
```

Comment:
Since you are fiddling with that I wonder if we can start testing against Ansible 2.2.2.0? In one of our projects we started using Ansible 2.2.2.0, which in actual fact uses this repo already.

For example:
```
env:
  - ANSIBLE_INSTALL_VERSION=2.0.2.0
  - ANSIBLE_INSTALL_VERSION=2.1.1.0
  - ANSIBLE_INSTALL_VERSION=2.2.2.0
```